src/CommandLineOptions.ts

```diff
@@ -35,6 +36,7 @@ export function mainCommand(
     .command('index')
     .option('--cwd <path>', 'the working directory', process.cwd())
     .option('--yarn-workspaces', 'whether to index all yarn workspaces', false)
+    .option('--yarn-berry-workspaces', 'whether to index all yarn v3 workspaces', false)
```

Comment:
@valerybugakov would it make sense to name this flag `--yarn-v3-workspaces`?

Reply:
@olafurpg, my rationale behind `berry` naming here is that this API was changed in the v2 and probably will be stable in v4 — it came with the `berry` rewrite of `yarn`. Open to other ideas, but tieing it to the specific `yarn` version can be misleading.

Reply:
Fine by me